frontend/src/components/pages/Scheduling/index.tsx

```diff
@@ -50,7 +50,7 @@ const schedulingDefaultData = ({
   size: "",
   dayPart: "",
   startTime: new Date().toDateString(),
-  endTime: new Date().toDateString(),
+  endTime: "",
```

Comment:
Setting this to empty string does not allow to select different date and it shows as invalid date. @AmandaMisjuwar do you know how we can change the date time logic so that this change does not affect how selecting date works?

Reply:
Why is it initialized to "" again?

Reply:
I set it to null so it would be easier to tell if the user actually selected a time range, can change this